parent.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE parent (
+    `id` INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    `firstname` VARCHAR(255) NOT NULL,
+    `lastname` VARCHAR(255) NOT NULL,
+    `email` VARCHAR(255) NOT NULL,
+    `phone_number` INT NOT NULL
```

Comment:
phone numbers should be treated as strings, never as numbers
it's quite a complicated matter because some people instinctively add dots or spaces between the numbers when typing their phone number, or add their region phone code at the beginning of the phone nulber (ex: +33 for France)